engine/src/cmd/weapon_xml.h

```diff
@@ -104,7 +104,7 @@ struct weapon_info
 private:
     float RefireRate;
 };
-weapon_info getWeaponInfoFromBuffer( char *netbuf, int &size );
+//weapon_info getWeaponInfoFromBuffer( char *netbuf, int &size );
```

Comment:
is this simply not used? We should add a comment here why it's disabled other than compiler warning cleanups.

Reply:
@Benjamin The text string "getWeaponInfoFromBuffer" occurs exactly twice in all of Vega Strike's source. Once in weapon_xml.h, where it prototypes the function, and once in weapon_xml.cpp where the function is defined. 

I think it's a leftover from the network code, as it is trying to read a netbuffer. 

I have actually removed that line completely in the next update that I am going to push when I have the next warning solved. So, come the next push that line will disappear completely and vex the next programmer no more.

I feel guilty of letting all those builds in Travis run for every small little update, so now I try to do the updates in batches.

Reply:
let's try to make that clear at least in the commit comments then you should be golden.